source/unified-test-format/unified-test-format.md

```diff
@@ -2939,12 +2941,20 @@ command to create the collection with the specified options. The test runner MUS
 topology is sharded, the test runner SHOULD use a single mongos for handling [initialData](#initialData) to avoid
 possible runtime errors.
 
+If the test might execute a sharded transaction, the test runner MUST collect the latest cluster time from processing
+[initialData](#initialData). See
+[MigrationConflict Errors on Sharded Clusters](#migrationconflict-errors-on-sharded-clusters) for more information.
+
 Create a new [Entity Map](#entity-map) that will be used for this test. If [createEntities](#createentities) is
 specified, the test runner MUST create each [entity](#entity) accordingly and add it to the map. If the topology is a
 sharded cluster, the test runner MUST handle [useMultipleMongoses](<>) accordingly if it is specified for any client
 entities. If the topology type is `LoadBalanced`, client entities MUST be initialized with the appropriate load balancer
 connection string as discussed in [useMultipleMongoses](<>).
 
+If the test might execute a sharded transaction, the test runner MUST advance the cluster time of any session entities
```

Comment:
Since `sharded transaction` leaves room for ambiguity (i.e. implying that multiple shards must be involved in the transaction), how about `execute a transaction against a sharded deployment` (here and below in a few other places)?